lib/context.js

```diff
@@ -71,6 +73,10 @@ function create(options) {
         context.data[k] = context.secrets[k] = options.secrets[k];
     }
 
+    for (let k in options.meta) {
+        context.data[k] = context.meta[k] = options.meta[k];
```

Comment:
I don't think we should put `meta` in `ctx.data`. I don't believe this is consistent w/ production.

Reply:
Removed